src/test/java/org/folio/consortia/controller/PublicationControllerTest.java

```diff
@@ -281,4 +286,17 @@ void parallelTenantRequestsTest(int tenantsAmount, int chunkSize) throws JsonPro
     // check if all 'x-okapi-tenant' values match with the initial list of expected tenant names
     Assertions.assertTrue(CollectionUtils.isEqualCollection(capturedTenantHeaders, listOfTenantNames));
   }
+
+  @Test
+  void publicationCleanup() throws Exception {
+    var headers = defaultHeaders();
+
+    when(consortiumRepository.findAll()).thenReturn(List.of(new ConsortiumEntity()));
+    when(publicationStatusRepository.deleteAllByCreatedDateBefore(any())).thenReturn(1);
+    when(publicationTenantRequestRepository.deleteAllByCreatedDateBefore(any())).thenReturn(2);
+
+    this.mockMvc.perform(post(String.format(PUBLICATIONS_CLEANUP_URL)).headers(headers))
+      .andExpectAll(status().is2xxSuccessful());
```

Comment:
maybe lets add mockito verify to tests that both dao methods invoked

Reply:
that would be redundant check. mockito will fail if when-then mocked method didnt invoke during the test